restli-client/src/main/java/com/linkedin/restli/client/ActionRequestBuilder.java

```diff
@@ -70,6 +71,11 @@ public ActionRequestBuilder(String baseUriTemplate, TypeSpec<V> elementType, Res
     _elementType = elementType;
   }
 
+  public ActionRequestBuilder<K, V> enableMutableActionParams(boolean enable) {
```

Comment:
On second thought, I am wondering since we introduce this flag for ActionRequestBuilder, we actually can provide two flavors of DynamicRecordTemplate.setValue based on a flag, safe vs unsafe, so that we don't change existing default behavior.

Reply:
I think this one is ok. The safety checks inside DynamicRecordTemplate weren't doing much anyways...